src/styles/StyleUtils.ts

```diff
@@ -513,34 +470,20 @@ function getIconFillColor(buttonState = CONST.BUTTON_STATES.DEFAULT, isMenuIcon
     }
 }
 
-/**
- * @param {Animated.Value} rotate
- * @param {Animated.Value} backgroundColor
- * @returns {Object}
- */
-function getAnimatedFABStyle(rotate, backgroundColor) {
+function getAnimatedFABStyle(rotate: Animated.Value, backgroundColor: Animated.Value): Animated.WithAnimatedValue<ViewStyle> {
     return {
         transform: [{rotate}],
         backgroundColor,
     };
 }
 
-/**
- * @param {Number} width
- * @param {Number | null} height
- * @returns {Object}
- */
-function getWidthAndHeightStyle(width, height = null) {
+function getWidthAndHeightStyle(width: number, height: number | null = null): ViewStyle | CSSProperties {
```

Comment:
Maybe let's use `undefined` to stay consistent?

Reply:
Good point